test/commands/generate/app_test.rb

```diff
@@ -45,6 +45,11 @@
       end
     end
 
+    it 'returns valid classified app name' do
+      command = Lotus::Commands::Generate::App.new({}, 'test-app')
```

Comment:
@davydovanton Can we use `"awesome-test-app"` instead?

I want to make sure that multiple dashes are handled well in the name. One day we could change `String#tr` with `String#sub` and the test still pass with `"test-app"`, but the feature would be broken.